src/ng/filter/orderBy.js

```diff
@@ -177,7 +177,7 @@ orderByFilter.$inject = ['$parse'];
 function orderByFilter($parse) {
   return function(array, sortPredicate, reverseOrder) {
 
-    if (!(isArrayLike(array))) return array;
+    if (!(isArrayLike(array))) throw minErr('filter')('notarray', 'Expected array but received: {0}', array);
```

Comment:
`grunt test` runs without errors if I change `minErr('filter')` to `minErr('orderBy')`